### PR TITLE
Set default heap memory and node name

### PR DESCRIPTION
**Pull request by mxm**
- updates also to Alpine Linux 3.6

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -66,4 +68,4 @@ WORKDIR /data
 EXPOSE 4200 4300 5432-5532
 
 ENTRYPOINT ["/docker-entrypoint.sh"]
-CMD ["crate"]
+CMD ["crate", "-Cnetwork.host=_site_"]
```
Comment (chaudum) [inline Dockerfile]: IMO this should be in the crate.yml

File: Dockerfile [hunk 2]
```diff
@@ -51,6 +51,8 @@ RUN apk add --no-cache --virtual .crate-rundeps \
     && apk del .build-deps
 
 ENV PATH /crate/bin:$PATH
+# Default heap size for Docker, can be overwritten by args
+ENV CRATE_HEAP_SIZE 1024M
```
Comment (chaudum) [inline Dockerfile]: all other distros have `512m` we should have it consistent

File: docker-entrypoint.sh [hunk 3]
```diff
@@ -8,7 +8,7 @@ fi
 
 if [ "$1" = 'crate' -a "$(id -u)" = '0' ]; then
     chown -R crate:crate /data
-    set -- gosu crate "$@"
+    set -- gosu crate:crate crate "$@"
```
Comment (chaudum) [inline docker-entrypoint.sh]: why did this change?

File: Dockerfile [hunk 4]
```diff
@@ -4,7 +4,7 @@
 # https://github.com/crate/docker-crate
 #
 
-FROM alpine:3.5
+FROM alpine:3.6
```
Comment (chaudum) [inline Dockerfile]: please upgrade alpine in a separate commit.

File: docker-entrypoint.sh (hunk 3 above)
Comment (mxm) [reply docker-entrypoint.sh]: `gosu` parameters changed

File: Dockerfile (hunk 1 above)
Comment (mxm) [reply Dockerfile]: as you wish

**Comment by mxm** [comment]
@chaudum separated the changes.

File: docker-entrypoint.sh (hunk 3 above)
Comment (chaudum) [reply docker-entrypoint.sh]: that's kinda strange, because gosu version did not change.

Comment (mxm) [reply docker-entrypoint.sh]: True, the version didn't change but I don't know how it every could have worked, these are the parameters for `gosu`:

```
Usage: gosu user-spec command [args]
   ie: gosu tianon bash
       gosu nobody:root bash -c 'whoami && id'
       gosu 1000:1 id

gosu version: 1.9 (go1.6.2 on linux/amd64; gc)
```

So the correct call is `gosu crate crate "$@"` or `gosu crate:crate crate "$@"`.

Comment (mxm) [reply docker-entrypoint.sh]: Ah, it passes the `crate` command from the args and only sets the crate user here. Sorry, see that now.

**Comment by mxm** [comment]
I don't know why the tests fail, I can run them locally without errors. Travis builds the Docker image but then fails to find it.

**Comment by chaudum** [comment]
@mxm maybe travis needs to build the image in the before_install section ?

**Comment by mxm** [comment]
@chaudum No idea why the Alpine linux commit broke the tests on Travis but reverting it let's the tests pass.

**Pull request merged**